### PR TITLE
remove react-dom dependency on React Native

### PR DESCRIPTION
**Pull request by nihgwu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,6 +102,9 @@
     "react": "global:React",
     "react-dom": "global:ReactDOM"
   },
+  "react-native" : {
```
Comment (sorrycc) [inline package.json]: 为啥加这段？给个来源链接吧

**Comment by nihgwu** [comment]
应该是这个以及下面的comments https://github.com/facebook/react-native/pull/2208

如果合并了能发个新版本吗，有代码洁癖😂

用dva开发react native有一段时间了，很棒的框架👍

能把我加到主维护者吗，我想进一步改进在RN中的开发体验，这两天会写一个更加贴近真实项目的demo，还有router相关的部分

On 8 Mar 2017, at 08:22, chencheng (云谦) <notifications@github.com<mailto:notifications@github.com>> wrote:


@sorrycc commented on this pull request.

________________________________

In package.json<https://github.com/dvajs/dva/pull/686#discussion_r104815892>:

> @@ -103,6 +102,9 @@
     "react": "global:React",
     "react-dom": "global:ReactDOM"
   },
+  "react-native" : {


为啥加这段？给个来源链接吧

**Comment by sorrycc** [comment]
@nihgwu  👍🏾

**Comment by nihgwu** [comment]
Thanks

On 8 Mar 2017, at 08:59, chencheng (云谦) <notifications@github.com<mailto:notifications@github.com>> wrote:


@nihgwu<https://github.com/nihgwu> 👍🏾

**Comment by nihgwu** [comment]
any plan to ship this? @sorrycc

**Comment by sorrycc** [comment]
+ dva@1.3.0-beta2

**Comment by nihgwu** [comment]
npm 上还是 1.2.1

**Comment by afc163** [comment]
```
npm dist-tag ls dva
beta: 1.3.0-beta2
latest: 1.2.1
```

**Comment by nihgwu** [comment]
@afc163 OK，看到了

还有貌似如果根据 semver 的规则，应该是 `1.3.0-beta.2` 而不是 `1.3.0-beta2` @sorrycc

File: package.json (hunk 1 above)
Comment (alsotang) [reply package.json]: https://github.com/facebook/react-native/pull/2208

Comment (alsotang) [reply package.json]: sorry，忘记看讨论区了。忽略我

**Pull request merged**